pkgs/top-level/ocaml-packages.nix

```
@@ -1701,7 +1701,9 @@ in let inherit (pkgs) callPackage; in rec
 
   ocamlPackages_5_0 = mkOcamlPackages (callPackage ../development/compilers/ocaml/5.0.nix { });
 
-  ocamlPackages_latest = ocamlPackages_5_0;
+  ocamlPackages_5 = ocamlPackages_5_0;
+
+  ocamlPackages_latest = ocamlPackages_5;
```

```suggestion
  ocamlPackages_latest = ocamlPackages_5;
  ocamlPackages_stable = ocamlPackages_4;
```
On the future, when default and latest points to same version, we can swap default to latest. So adding `stable` in advance makes sense for people that actually wants the stable version.